src/DynamicMvvm.Abstractions/ViewModel/IViewModel.Extensions.Commands.cs

```diff
@@ -79,6 +79,37 @@ public static partial class IViewModelExtensions
 			[CallerMemberName] string name = null
 		) => viewModel.GetOrCreateCommand(name, n => viewModel.GetDynamicCommandBuilderFactory().CreateFromTask(n, execute), configure);
 
+		/// <summary>
+		/// Gets or creates a unique instance of <see cref="IDynamicCommand"/> configured for every items in a list
+		/// with base name + guid.
+		/// </summary>
+		/// <param name="viewModel">The ViewModel.</param>
+		/// <param name="execute">The execute method.</param>
+		/// <param name="configure">The optional function to configure the command builder.</param>
+		/// <param name="name">The command name.</param>
+		/// <returns><see cref="IDynamicCommand"/></returns>
+		public static IDynamicCommand GetItemCommandFromTask(
+			this IViewModel viewModel, Func<CancellationToken, Task> execute,
+			Func<IDynamicCommandBuilder, IDynamicCommandBuilder> configure = null,
+			[CallerMemberName] string name = null
+			) => viewModel.GetCommandFromTask(execute, configure: c => c, name: (name + Guid.NewGuid()));
+
+		/// <summary>
+		/// Gets or creates a unique instance of <see cref="IDynamicCommand"/> configured for every items in a list.
+		/// with base name + guid.
+		/// </summary>
+		/// <typeparam name="TParameter">The type of the command parameter.</typeparam>
+		/// <param name="viewModel">The ViewModel.</param>
+		/// <param name="execute">The execute method.</param>
+		/// <param name="configure">The optional function to configure the command builder.</param>
+		/// <param name="name">The command name.</param>
+		/// <returns><see cref="IDynamicCommand"/></returns>
+		public static IDynamicCommand GetItemCommandFromTask<TParameter>(
+			this IViewModel viewModel, Func<CancellationToken, TParameter, Task> execute,
+			Func<IDynamicCommandBuilder, IDynamicCommandBuilder> configure = null,
+			[CallerMemberName] string name = null
```

Comment:
this parameter could be renamed to `namePrefix` because that's what it really is